specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-11-01/netapp.json

```diff
@@ -0,0 +1,4873 @@
+        {
+          "$ref": "#/parameters/SubscriptionId"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroup"
+        },
+        {
+          "$ref": "#/parameters/AccountName"
+        },
+        {
+          "$ref": "#/parameters/SnapshotPolicyName"
+        },
+        {
+          "$ref": "#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "Snapshot Policy"
+        ],
+        "operationId": "SnapshotPolicies_Get",
+        "description": "Get a snapshot Policy",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "SnapshotPolicies_Get": {
+            "$ref": "examples/SnapshotPolicies_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Snapshot Policy"
+        ],
+        "operationId": "SnapshotPolicies_Create",
+        "description": "Create a snapshot policy",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "Snapshot policy object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - snapshot Policy updated",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          },
+          "201": {
+            "description": "Snapshot Policy created",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": false,
+        "x-ms-examples": {
+          "SnapshotPolicies_Create": {
+            "$ref": "examples/SnapshotPolicies_Create.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Snapshot Policy"
+        ],
+        "operationId": "SnapshotPolicies_Update",
+        "description": "Patch a snapshot policy",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "Snapshot policy object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicyPatch"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicy"
+            }
+          },
+          "202": {
```

Comment:
Please set `x-ms-long-running-operation` and `x-ms-long-running-operation-options` since it is async.

https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation
RPC: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#updating-using-patch

Reply:
We have `x-ms-long-running-operation` but they do not have  `x-ms-long-running-operation-options`.   Should we apply `x-ms-long-running-operation-options`  to them all as well?